release/src/version-helpers.ts

```diff
@@ -206,13 +214,42 @@ export async function getLastEmbeddingSdkReleaseTag({
     ref: `tags/embedding-sdk-0.${majorVersion}`,
   });
 
+  const versions = _.compose(
+    filterOutNonSupportedPrereleaseIdentifiers,
+    getVersionsFromTags,
+  )(tags);
+
   const lastRelease = getLastReleaseFromTags({
-    tags,
+    tags: versions,
```

Comment:
Here's the fix. We filter out tags/versions that have a  prerelease identifier (the one after the dash in `0.55.5-metabot`) that we don't support. Currently we only allow normal versions without the prerelease identifier, and `-nightly`.